package.json

```diff
@@ -99,6 +100,7 @@
     "file-loader": "^3.0.1",
     "html-webpack-plugin": "^3.2.0",
     "husky": "^1.3.1",
+    "hygen": "^4.0.7",
```

Comment:
It looks like you didn't add the lockfile so I get an error when I run `yarn bootstrap`
```bash
dna Civic: (feature/template-api-card)$ yarn bootstrap
yarn run v1.16.0
$ yarn install --frozen-lockfile && yarn --cwd ./packages/component-library/ build && yarn --cwd ./packages/mock-wrapper/ build
[1/4] 🔍  Resolving packages...
error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
I added the lockfile which resolves this error.

Reply:
@BJHunnicutt, make sure to add the `yarn.lock` that's generated whenever you add something to `package.json`. That file makes sure that every developer is using the correct version of dependencies and that those dependencies can be effectively symlinked by yarn workspaces